src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -1425,6 +1439,7 @@ protected void validatePlatform(
                       "property '%s' value was not a valid integer: %s",
                       property.getName(), property.getValue()));
         }
+      } else if (validForOperationQueue) {
```

Comment:
add a comment here that we're skipping things and that they are valid, no action is correct here

Reply:
fixed